mslib/msui/editor.py

```diff
@@ -51,22 +56,20 @@ def __init__(self, parent=None):
         self.editor = QtWidgets.QPlainTextEdit()
 
         # Load mss_settings.json (if already exists)
-        self.mss_path = fs.path.join(MSS_CONFIG_PATH, "mss_settings" + ".json")
-        if fs.open_fs(MSS_CONFIG_PATH).exists("mss_settings.json"):
-            file_name = fs.path.basename(self.mss_path)
-            with fs.open_fs(fs.path.dirname(self.mss_path)) as file_dir:
-                self.file_content = file_dir.readtext(file_name)
+        self.path = constants.CACHED_CONFIG_FILE
+        if self.path and os.path.exists(self.path):
```

Comment:
with using os self.path can't be a FS Url. This should be replaced by a https://docs.pyfilesystem.org/en/latest/reference/base.html#fs.base.FS.exists

Reply:
So I have to first split the name in directory, and filename, open the directory as file system and then for the existance of the filename in the file system? That seemed a bit convoluted to me.

Reply:
I have implemented an fs-based replacement, but the new code is way less stable than the previous one. I do not know fs well enough to make it better.
E.g. can the "split" command fail? What is when the "dirname" cannot be opened?

Reply:
If you have selected a file name in a browser just before someone changes the dir or the rights of that, this gives other problems. 
I will later do a review of all fs used lines, that can be done in this #327